new/LIT7074AynatTsella.xml

```diff
@@ -0,0 +1,86 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7074AynatTsella" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">አድኅነኒ፡ እምሕማመ፡ ዓይነ፡ ጽላ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾAdḫǝnani ʾǝm-ḥǝmāma ʾayna ṣǝlā ...</title>
```

Comment:
I would transcribe ʾAdḫǝnanni. And if we interpret Ṣǝlā as a proper noun, then maybe we could capitalize it in the transcription as well?